app/controllers/application_controller.rb

```diff
@@ -1,8 +1,15 @@
 # Top-level documentation comment for ApplicationController module
 class ApplicationController < ActionController::Base
-  helper_method :current_user
+  before_action :configure_permitted_parameters, if: :devise_controller?
+  before_action :authenticate_user!
 
-  def current_user
-    @current_user = User.first
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.permit(:sign_up, keys: [:name])
+  end
```

Comment:
**[OPTIONAL]**
  I am making this change optional because the previous reviewer missed it and I think it's important to address. Kindly make this change before you merge your work.

- [ ] It will be great to permit all the parameters needed when you are signing up as a new user. It seems you only permitted the name and hence there is an error in the terminal. Below is the error in the terminal ⬇️ 
<img width="1982" alt="Screenshot 2023-05-19 at 04 12 25" src="https://github.com/Uthmanbello/myblog/assets/32361963/e5ff63cd-f507-4aab-9b2a-ff897c871874">

Reply:
@peterrobert ,

Thanks a lot